src/tests/functional/form_mode_tests.js

```diff
@@ -65,11 +65,11 @@ test("test form submission with form mode optin and form enabled from submitter
   assert.ok(await formSubmitStarted(page))
 })
 
-async function gotoPageWithFormMode(page: Page, formMode: "on" | "off" | "optin") {
+async function gotoPageWithFormMode(page, formMode) {
```

Comment:
@vfonic how do you know it can be one of those three? It looks like it can be any primitive, object, null or undefined?

Reply:
@pailhead because it was typed like this before 🤷‍♂️?

Reply:
That's the point. That information is now gone, so now it can be anything.

I think they want to rely on tests now only to catch these misuses. Hopefully there is a test for this.

Reply:
@Wirone it is a large PR and there is no type to narrow it. Without seeing who is calling it I don’t think it’s possible to narrow it down to even a string, let alone these three particular ones.